x/params/keeper/common_test.go

```diff
@@ -1,16 +1,25 @@
-// nolint:deadcode,unused
-package params
+package keeper
 
 import (
-	abci "github.com/tendermint/tendermint/abci/types"
-	"github.com/tendermint/tendermint/libs/log"
-	dbm "github.com/tendermint/tm-db"
-
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/store"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	dbm "github.com/tendermint/tm-db"
 )
 
+func testComponents() (codec.Marshaler, sdk.Context, sdk.StoreKey, sdk.StoreKey, Keeper) {
+	cdc := createTestCodec()
+	mkey := sdk.NewKVStoreKey("test")
+	tkey := sdk.NewTransientStoreKey("transient_test")
+	ctx := defaultContext(mkey, tkey)
+	keeper := NewKeeper(cdc, mkey, tkey)
+
+	return cdc, ctx, mkey, tkey, keeper
+}
+
 type invalid struct{}
 
 type s struct {
```

Comment:
remove this as well of `invalid` type

Reply:
it is not descriptive of what it does

Reply:
Tried it already, but coupled to all the tests due to testComponents() function. My suggestion is to avoid in this PR to refactor tests and create an issue or when we migrate this module to simapp to rewrite those tests and remove all this things.